### DISCUSSION TITLE
OctoPrint - plugins fail to install

### DISCUSSION THREAD

**Discussion by smarthomelawyer** [tteck/Proxmox#1384] [Help Needed]
First, thank you so much for the the OctoPrint LXC! 

Install went smoothly, as usual, but I seem to be running into issues installing Plugins. No matter which one I try to install, I get the below error (this one is the OctoPod plugin). Any ideas on what I'm doing wrong?

![image](https://user-images.githubusercontent.com/74792554/235704173-5b80e10d-bf81-471e-b8f3-3504e5d05e1e.jpeg)

Installing plugin "OctoPrint-OctoPod" from https://github.com/gdombiak/OctoPrint-OctoPod/archive/master.zip...
/opt/octoprint/bin/python3 -m pip --disable-pip-version-check install file:///tmp/tmp260c6xzx/OctoPrint-OctoPod-master.zip --no-cache-dir
Processing /tmp/tmp260c6xzx/OctoPrint-OctoPod-master.zip
Installing build dependencies: started
error: subprocess-exited-with-error
× Getting requirements to build wheel did not run successfully.
│ exit code: 255
╰─> [1 lines of output]
Could not import OctoPrint's setuptools, are you sure you are running that under the same python installation that OctoPrint is installed under?
[end of output]
note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error
× Getting requirements to build wheel did not run successfully.
│ exit code: 255
╰─> See above for output.
note: This error originates from a subprocess, and is likely not a problem with pip.
Installing build dependencies: finished with status 'done'
Getting requirements to build wheel: started
Getting requirements to build wheel: finished with status 'error'
Error!
Could not parse output from pip, see plugin_pluginmanager_console.log for generated output

**Comment by smarthomelawyer** [comment]
Looks like an issue with OctoPrint itself - switching to OctoPrint 1.9.0 (currently in RC testing phase) solves the issue. 

 For anyone with same issue, more info on the fix here: 

(https://community.octoprint.org/t/problem-cant-install-3rd-party-plugins/51349)

**Reply by tteck** [reply]
I'll add `pip install wheel`
